package.json

```diff
@@ -46,6 +47,7 @@
     "es6-dom": "^1.0.0",
     "grunt": "^0.4.5",
     "grunt-babel": "^6.0.0",
+    "grunt-cli": "^1.2.0",
```

Comment:
otherwise, you're relying on people having the right global version of grunt-cli install (i had an old version, and was greeted with an error when trying to build).